src/Models/ProductList.tsx

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
`ProductsList` component should be inside components folder. Use `Models` for interfaces/types